### PR TITLE
fix: correctly sample evals

### PR DESCRIPTION
**Pull request by maxdeichmann**

### CODE REVIEW

File: worker/src/__tests__/eval-service.test.ts [hunk 1]
```diff
@@ -199,6 +199,104 @@ describe("create eval jobs", () => {
...
+      data: {
+        id: randomUUID(),
+        projectId: "7a88fb47-b4e2-43b8-a06c-a5ce950dc53a",
+        filter: JSON.parse("[]"),
+        jobType: "EVAL",
+        delay: 0,
+        sampling: new Decimal("0"),
+        targetObject: "traces",
+        scoreName: "score",
+        variableMapping: JSON.parse("[]"),
```
Comment (marliessophie) [inline worker/src/__tests__/eval-service.test.ts:280]: Nit: pass status: `ACTIVE` for clarity

File: worker/src/__tests__/eval-service.test.ts [hunk 2]
```diff
@@ -199,6 +199,104 @@ describe("create eval jobs", () => {
...
+    const jobs = await kyselyPrisma.$kysely
+      .selectFrom("job_executions")
+      .selectAll()
+      .where("project_id", "=", "7a88fb47-b4e2-43b8-a06c-a5ce950dc53a")
+      .execute();
+
+    expect(jobs.length).toBe(0);
+  }, 10_000);
+
+  test("does not create eval job for 0 sample rate", async () => {
```
Comment (marliessophie) [inline worker/src/__tests__/eval-service.test.ts:245]: Nit: Shall we add another test for a sample rate of say 50%?

**Review by marliessophie — APPROVED** [review]
Change makes sense to me, thank you! Note, there seems to be one failing test though.

File: worker/src/__tests__/eval-service.test.ts (hunk 2 above)
Comment (maxdeichmann) [reply worker/src/__tests__/eval-service.test.ts:245]: also thought about this but hard to assert. I would not do it now.

**Pull request merged**